common/lib/xmodule/xmodule/js/src/capa/display.coffee

```diff
@@ -437,15 +441,37 @@ class @Problem
     if not @check_save_waitfor(@save_internal)
       @disableAllButtonsWhileRunning @save_internal, false
 
+  focus_on_save_notification: =>
+    @saveNotificationArea = @$('.notification.save')
+    @saveNotificationArea.focus()
+    # TODO: Wait for render to complete to disable the save button and focus.
+    @enableSaveButton false, true
+
   save_internal: =>
     Logger.log 'problem_save', @answers
     $.postWithPrefix "#{@url}/problem_save", @answers, (response) =>
-      saveMessage = response.msg
       if response.success
         @el.trigger('contentChanged', [@id, response.html])
-      @gentle_alert saveMessage
+        @render(response.html, @focus_on_save_notification)
```

Comment:
Note that this loses the display of any possible errors. If there is a server-side error on save, the message about the error will be in response.msg (see save_problem method in capa_base.py).
 So the code should still call "gentle_alert" (which will be rewritten in a future story) if response.success is False.